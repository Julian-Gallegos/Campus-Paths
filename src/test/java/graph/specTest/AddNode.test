# Tests that a node can be added to a graph

#Create an empty graph
CreateEmptyGraph graph1

# Add node to a graph
AddNode graph1 node1