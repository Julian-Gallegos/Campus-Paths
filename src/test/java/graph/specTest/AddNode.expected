# Tests that a node can be added to a graph

#Create an empty graph
created graph graph1

# Add node to a graph
added node node1 to graph1